packages/fields-location-google/src/test-fixtures.js

```diff
@@ -1,9 +1,216 @@
+      ])
+    )
+  );
+};
+
+export const crudTests = withKeystone => {
+  const withHelpers = wrappedFn => {
+    return async ({ keystone, listKey }) => {
+      const items = await getItems({
+        keystone,
+        listKey,
+        returnFields: 'id venue { googlePlaceID }',
+        sortBy: 'name_ASC',
+      });
+
+      return wrappedFn({ keystone, listKey, items });
+    };
+  };
+
+  test(
+    'Create',
+    withKeystone(
+      withHelpers(async ({ keystone, listKey }) => {
+        const data = await createItem({
+          keystone,
+          listKey,
+          item: { name: '', venue: 'ChIJQaHdVx-uEmsRk0z70ZfPYok' },
+          returnFields: 'id venue { googlePlaceID }',
+        });
+        expect(data).not.toBe(null);
+        expect(data.venue.googlePlaceID).toEqual('ChIJQaHdVx-uEmsRk0z70ZfPYok');
+      })
+    )
+  );
+
+  test(
+    'Read',
+    withKeystone(
+      withHelpers(async ({ keystone, listKey, items }) => {
+        const data = await getItem({
+          keystone,
+          listKey,
+          itemId: items[0].id,
+          returnFields: 'id venue { googlePlaceID }',
+        });
+
+        expect(data).not.toBe(null);
+        expect(data.venue.googlePlaceID).toBe(items[0].venue.googlePlaceID);
+      })
+    )
+  );
+
+  describe('Update', () => {
+    test(
+      'Updating the value',
+      withKeystone(
+        withHelpers(async ({ keystone, items, listKey }) => {
+          const data = await updateItem({
+            keystone,
+            listKey,
+            item: {
+              id: items[0].id,
+              data: { venue: 'ChIJeRC1LkiuEmsRAiqhKVgHRMQ' },
+            },
+            returnFields: 'venue { googlePlaceID }',
+          });
+          expect(data).not.toBe(null);
+          expect(data.venue.googlePlaceID).toBe('ChIJeRC1LkiuEmsRAiqhKVgHRMQ');
+        })
+      )
+    );
+
+    test(
+      'Updating the value to null',
+      withKeystone(
+        withHelpers(async ({ keystone, items, listKey }) => {
+          const data = await updateItem({
+            keystone,
+            listKey,
+            item: {
+              id: items[3].id,
+              data: { venue: null },
+            },
+            returnFields: 'venue',
+          });
+          expect(data).not.toBe(null);
+          expect(data.venue).toBe(null);
+        })
+      )
+    );
+
+    test(
+      'Updating without this field',
+      withKeystone(
+        withHelpers(async ({ keystone, items, listKey }) => {
+          const data = await updateItem({
+            keystone,
+            listKey,
+            item: {
+              id: items[0].id,
+              data: { name: 'New location' },
+            },
+            returnFields: 'name venue { googlePlaceID }',
+          });
+          expect(data).not.toBe(null);
+          expect(data.name).toBe('New location');
+          expect(data.venue.googlePlaceID).toBe(items[0].venue.googlePlaceID);
+        })
+      )
+    );
+  });
```

Comment:
Should our CRUD tests on this (and other fields) include a `delete` operation?

Reply:
Hmm. That's a good point. I think we missed having `delete` test. I shall go ahead and add those too. 👍